tests/ssl/openssl.cnf

```diff
@@ -14,10 +14,30 @@ distinguished_name = req_dn
 basicConstraints = CA:false
 keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 subjectAltName = DNS:localhost
+authorityInfoAccess = OCSP;URI:http://127.0.0.1:8888
+crlDistributionPoints = URI:http://127.0.0.1:8889/crl.pem
```

Comment:
I don't think I'm the "shad" you're looking for 😬 .  Perhaps @shadinua ?

Reply:
Oh, *that* why there is no response 😂

I'm terribly sorry for the mistake. But well... welcome to our pull request 😁

Reply:
> Isn't localhost better? Like, for built-in IPv6 support.
Of course, in general case it is preferable to use domain names instead of IPs when it possible.

But. During a protracted IPv4 → IPv6 transition period that multiplied by a rather complex resolving mechanism in *nix systems, sometimes we can observe a lot of strange side effects, when `localhost` may not be functionally equal neither to `127.0.0.1` nor to `::1`.

So I wouldn't say that using `localhost` instead of `127.0.0.1` make things better in that particular case.

Generally speaking I'd suggest to use:
* `localhost`/`localhost6` — when we either do production-ready installations or we want to test not only the product, but the whole interaction with the base OS
* `127.0.0.1` — when we want to test exceptionally the product and be sure that the [mis]configuration resolving subsystem on base OS will not affect on the flow; today, by default, socket on `127.0.0.1` will be created even if we specify `::1` as IP to bind, so in observable future this approach should work; as a bonus — this approach is a bit faster that also may be useful in tests